ui/dasher/src/dasher.ts

```diff
@@ -77,7 +77,7 @@ export function makeCtrl(opts: DasherOpts, data: DasherData, redraw: Redraw): Da
 
   function toggleZen() {
     data.zen = data.zen ? 0 : 1;
```

Comment:
Does this still make sense?

Reply:
yep, not anymore

Reply:
i double checked and then did this: d418db5d3a288a3996962b81bd131797b7621303